Week1/arrays.js

```diff
@@ -1,8 +1,8 @@
 //append arrays
 let arraysInside = [];
-console.log ('my array will contain: eagle, horse and dog');
+console.log ('my array will contain some animals');
 console.log(arraysInside);
 let myFavoriteAnimals = ['eagle ', 'dog ', 'horse '];
 console.log(myFavoriteAnimals);
-arraysInside = ['baby pig ' , myFavoriteAnimals];
+arraysInside.push('baby pig');
```

Comment:
It is almost correct! If you change `arraysInside.push('baby pig');` to `myFavoriteAnimals.push('baby pig');` then it is correct:)

Reply:
Understand. it's done. Thank you